libs/blocks/global-navigation/global-navigation.js

```diff
@@ -223,7 +327,8 @@ class Gnav {
           ${this.decorateBrand()}
         </div>
         ${this.elements.navWrapper}
-        ${this.blocks.profile.rawElem ? this.blocks.profile.decoratedElem : ''}
+        ${this.useUniversalNav ? this.blocks.universalNav : ''}
+        ${(!this.useUniversalNav && this.blocks.profile.rawElem) ? this.blocks.profile.decoratedElem : ''}
```

Comment:
If you're deleting `this.blocks.profile`, should we add some optional chaining here for the `rawElem`?

Reply:
We can do it to be super safe, but profile is deleted only if `useUniversalNav` is true